vms/platformvm/deposit/camino_deposit.go

```diff
@@ -60,6 +63,42 @@ func (o *Offer) InterestRateFloat64() float64 {
 	return float64(o.InterestRateNominator) / float64(interestRateDenominator)
 }
 
+func (o *Offer) Verify() error {
+	if o.Start >= o.End {
+		return fmt.Errorf(
+			"deposit offer starttime (%v) is not before its endtime (%v)",
+			o.Start,
+			o.End,
+		)
+	}
+
+	if o.MinDuration > o.MaxDuration {
```

Comment:
I guess our files have `Min == Max` or here it's fine?
```suggestion
	if o.MinDuration >= o.MaxDuration {
```

Reply:
No error f Min <= Max  -> thats fine

Reply:
oh - ur right!